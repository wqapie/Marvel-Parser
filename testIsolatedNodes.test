# Test isolated nodes (in different books)

LoadGraph g isolated.tsv

ListNodes g