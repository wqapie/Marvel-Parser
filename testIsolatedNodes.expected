# Test isolated nodes (in different books)

loaded graph g

g contains: Robin3 Robin666